backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -217,10 +217,15 @@ function replaceIngressApiServer (client, { name = TERMINAL_KUBE_APISERVER, name
           paths: [
             {
               backend: {
-                serviceName,
-                servicePort: 443
+                service: {
+                  name: serviceName,
+                  port: {
+                    number: 443
+                  }
+                }
               },
-              path: '/'
+              path: '/',
+              pathType: 'ImplementationSpecific'
```

Comment:
```suggestion
              pathType: 'Prefix'
```
We want a clearly defined pathType. It should not be implementation specific up to the IngressClass.

> - **ImplementationSpecific:** With this path type, matching is up to the IngressClass. Implementations can treat this as a separate pathType or treat it identically to Prefix or Exact path types.
> - **Exact:** Matches the URL path exactly and with case sensitivity.
> - **Prefix:** Matches based on a URL path prefix split by /. Matching is case sensitive and done on a path element by element basis. A path element refers to the list of labels in the path split by the / separator. A request is a match for path p if every p is an element-wise prefix of p of the request path.